src/D2DLibExport/D2DEnums.cs

```diff
@@ -345,7 +345,28 @@ public enum D2DBitmapInterpolationMode
 		//
 		Linear = 1,
 	}
-
+ 
+	public enum D2DExtendMode
+	{
+		/// 
+		/// Extend the edges of the source out by clamping sample points outside the source
+		/// to the edges.
+		///
```

Comment:
Using `<summary>` will allow these docs to appear in IDE tooling etc.

```suggestion
		/// <summary>
		/// Extend the edges of the source out by clamping sample points outside the source
		/// to the edges.
		/// </summary>
```

Ditto below.